components/automate-ui/src/app/page-components/search-bar/search-bar.component.ts

```diff
@@ -71,7 +71,8 @@ export class SearchBarComponent implements OnChanges {
   ngOnChanges(changes: SimpleChanges) {
     if (changes.dynamicSuggestions) {
       this.suggestions = List<SuggestionItem>(
-        compact(changes.dynamicSuggestions.currentValue.map((suggestion) => {
+        compact(changes.dynamicSuggestions.currentValue.filter(
+          (suggestion) => suggestion && suggestion.length !== 0).map((suggestion) => {
```

Comment:
Removing empty suggestion passed to the search bar component

Reply:
If you used lodash, you might be able to simplify this a bit like `.filter(isEmpty)`.